### DISCUSSION TITLE
Nullable<T> vs Maybe<T>

### DISCUSSION THREAD

**Discussion by xavierjohn** [vkhorikov/CSharpFunctionalExtensions#493] [Q&A]
Is there a reason to use Maybe<T> anymore compared to Nullable<T>?
We could add a ToResult extension to Nullable<T>.

**Comment by xavierjohn** [comment]
I decided to ditch Maybe since it did not feel natural in VS 2022. Maybe was a good option in the past.
`May be` we can conditionally add something like https://github.com/xavierjohn/FunctionalDDD/blob/main/RailwayOrientedProgramming/src/Result/Extensions/ToResult.cs where nullability is supported.

**Comment by vkhorikov** [comment]
`Nullable` only allows structs as type arguments.

**Comment by xavierjohn** [comment]
This PR will add ToResult for Null & Nullability.
https://github.com/vkhorikov/CSharpFunctionalExtensions/pull/494

Some of the Target Framework seems redundant so I have removed it. .NET Framework users can use the library that targets .NET standard 2.0.
